engine/redis_v2/timer.go

```diff
@@ -13,17 +15,22 @@ local zset_key = KEYS[1]
 local output_queue_prefix = KEYS[2]
 local pool_prefix = KEYS[3]
 local output_deadletter_prefix = KEYS[4]
-local now = ARGV[1]
+local min_score= ARGV[1]
 local limit = ARGV[2]
 
-local expiredMembers = redis.call("ZRANGEBYSCORE", zset_key, 0, now, "LIMIT", 0, limit)
+local expiredMembers = redis.call("ZRANGEBYSCORE", zset_key, 0, min_score, "WITHSCORES", "LIMIT", 0, limit)
 
 if #expiredMembers == 0 then
 	return 0
 end
 
-for _,v in ipairs(expiredMembers) do
-	local ns, q, tries, job_id = struct.unpack("Hc0Hc0HHc0", v)
+local toBeRemovedMembers = {}
```

Comment:
Is it necessary to add lines of comments for this block of code?

Reply:
Yes, will add more comments for them to make it easier to understand